recipes/pycollage/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pycollage" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycollage-{{ version }}.tar.gz
+  sha256: d80f2b638626e9d419862aa3ede3153ecbbb616e786a678c5cb07533bf5a3600
+
+build:
+  noarch: python
+  # Fix for windows described here:
+  # https://github.com/conda/conda-build/issues/2627#issuecomment-579377667
```

Comment:
No need for this comment here. We should always be explicit with the entry-points anyway.
```suggestion
```